templates/jitsi_meet_config.js.j2

```diff
@@ -170,7 +170,7 @@ var config = {
     desktopSharingChromeExtId: '{{ jitsi_meet_desktop_sharing_chrome_ext_id }}',
 
     // Whether desktop sharing should be disabled on Chrome.
-    desktopSharingChromeDisabled: {{ jitsi_meet_desktop_sharing_chrome_disabled }},
+    desktopSharingChromeDisabled: {{ not ( jitsi_meet_enable_desktop_sharing_chrome | bool ) | lower }},
```

Comment:
Here it should be `{{ (not jitsi_meet_enable_desktop_sharing_chrome) | bool | lower }}`, otherwise the result is `False` instead of `false` and config.json can't be parsed correctly